services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java

```diff
@@ -0,0 +1,334 @@
+   */
+  @AutoValue.Builder
+  public abstract static class Builder {
+
+    private Integer[] contoursMinutesArray;
+
+    /**
+     * Optionally change the APIs base URL to something other then the default Mapbox one.
+     *
+     * @param baseUrl base url used as end point
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public abstract Builder baseUrl(@NonNull String baseUrl);
+
+    /**
+     * A valid Mapbox access token.
+     *
+     * @param accessToken the Mapbox access token to use for the Isochrone API call
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public abstract Builder accessToken(@NonNull String accessToken);
+
+
+    /**
+     * A Mapbox Directions routing profile ID. Options are
+     * {@link IsochroneCriteria#PROFILE_DRIVING} for travel times by car,
+     * {@link IsochroneCriteria#PROFILE_WALKING} for pedestrian and hiking travel times,
+     * and {@link IsochroneCriteria#PROFILE_CYCLING} for travel times by bicycle.
+     *
+     * @param profile a Mapbox Directions profile
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public abstract Builder profile(@Nullable @IsochroneCriteria.IsochroneProfile String profile);
+
+    /**
+     * A {@link Point} object which represents a {longitude,latitude} coordinate
+     * pair around which to center the isochrone lines.
+     *
+     * @param queryPoint center query point for the isochrone calculation
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public Builder coordinates(@NonNull Point queryPoint) {
+      coordinates(String.format(Locale.US, "%s,%s",
+        TextUtils.formatCoordinate(queryPoint.longitude()),
+        TextUtils.formatCoordinate(queryPoint.latitude())));
+      return this;
+    }
+
+    /**
+     * A string which represents a {longitude,latitude} coordinate pair
+     * around which to center the isochrone lines. The String should be
+     * "longitude,latitude".
+     *
+     * @param queryPoint center query point for the isochrone calculation
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public abstract Builder coordinates(@NonNull String queryPoint);
+
+    /**
+     * An integer list of minute values to use for each isochrone contour.
+     * You must pass in at least one minute amount and you can specify
+     * up to four contours. Times must be in increasing order. For example,
+     * "5,20,40,50" and not "20,10,40,55". The maximum time that can be
+     * specified is 60 minutes.
+     *
+     * @param listOfMinuteValues an integer list with at least one number
+     *                           for the minutes which represent each contour
+     * @return this builder for chaining options together
+     * @since 4.6.0
+     */
+    public Builder contoursMinutes(@NonNull List<Integer> listOfMinuteValues) {
```

Comment:
Here is an example how it is done in other parts of mapbox-java. I think it would be good to follow the same pattern. https://github.com/mapbox/mapbox-java/blob/4afbd732b758010a70c742dbe3a3f15a9b7d13a7/services-matching/src/main/java/com/mapbox/api/matching/v5/MapboxMapMatching.java#L639

Reply:
Fixed 👍